### DISCUSSION TITLE
Latest wagmi issues with useContractRead

### DISCUSSION THREAD

**Discussion by dev-cz** [wevm/wagmi#1218] [Q&A]
Updated to the latest wagmi, and now none of my useContractRead hooks work, where previously they did. I made the changes, but can't seem to figure out whats wrong and how to fix. Any ideas?
![image](https://user-images.githubusercontent.com/51761060/199326891-3305af77-7bdb-4e8e-a3aa-c183b08911aa.png)

![image](https://user-images.githubusercontent.com/51761060/199327267-3c12691a-dbc0-4dd7-8d62-ef8673357818.png)

**Comment by tmm** [comment]
What version of wagmi are you using? Before version 0.7, `useContractRead` `address` was named `addressOrName`.

**Reply by dev-cz** [reply]
> What version of wagmi are you using? Before version 0.7, `useContractRead` `address` was named `addressOrName`.

So this new error was after upgrading to the latest 7.x version. Could not figure out how to get it working for the life of me.

So had to revert back to 6.x and change it back to addressOrName.

**Reply by tmm** [reply]
Can you share the code that is related to the error for `useContractRead`? All I see is the error message.

**Reply by dev-cz** [reply]
> Can you share the code that is related to the error for `useContractRead`? All I see is the error message.

So these are 2 examples of issues, where just renaming to 'address' did not work.

![image](https://user-images.githubusercontent.com/51761060/199557859-3c7d2b82-5e0c-4ae7-8339-8db0359f2d29.png)
![image](https://user-images.githubusercontent.com/51761060/199557881-8b79f13e-fd30-45d6-a403-49d1b760437f.png)

**Reply by tmm** [reply]
Can you provide some run-able code via CodeSandbox or a GitHub repo/gist? It's difficult to debug from screenshots alone.

**Reply by dev-cz** [reply]
> Can you provide some run-able code via CodeSandbox or a GitHub repo/gist? It's difficult to debug from screenshots alone.

Sure. Will be here in a bit. It is using react/typescript if that helps at all. But will try to get this going here on Codesandbox in a bit.
